res/css/views/right_panel/_ThreadPanel.pcss

```diff
@@ -130,7 +130,7 @@ limitations under the License.
     }
 
     .mx_MessageTimestamp {
-        color: $event-timestamp-color;
+        color: var(--MessageTimestamp-color); /* TODO: check whether needed or not */
```

Comment:
It is not sure if this declaration would be really required. After having more extensive test sets which check visual regressions overall, it might be fine to remove it.